### ISSUE TITLE
Problem with draft akdup

### ISSUE THREAD

**Issue by JanGen** [freesewing/site#259]
Sleeves seem to come out much too short (maybe not taking shoulder length into account?).
See [here](https://freesewing.org/drafts/akdup).

**Comment by joostdecock** [comment]
Thank you for reporting your issue. Let's see what's going on.

Pardon me if this is a silly question, but did you take into account that there is ribbing to be added to the cuffs?

Also, and this might not make a difference, I noticed your across back measurement is wrong. It's higher than the shoulder to shoulder which is not possible and seems to indicate you measured around the curvature of your back into the sides.

Please check the instructions here: https://freesewing.org/docs/measurements/acrossback

Thanks for clarifying these things for me.

**Comment by JanGen** [comment]
Thank you for responding so fast!

I corrected the across back measurement, I believe I misstyped (might still be slightly off, since I had to recheck on my own).

I looked at the comparison preview, which indicates that the sleeves are far too short. I also quickly crosschecked with the sleeve length of a similar garment that I own, which also indicates that at least a few centimeters are missing. I'm aware of the ribbing and took that into account.

I haven't printed the pattern yet or cut any fabric, so all of this relies on looking at the pattern and eye-balling what that would mean.

**Comment by joostdecock** [comment]
I'm glad you reached out. I will have a closer look at this when I can get to my laptop. I'll get back to you.

**Comment by joostdecock** [comment]
Ok, I've had a closer look at this. Here's what I've found:

 - From the side of your neck to the wrist, the length of the sleeve is 24.24 + 45.75 + 8 = 78cm
 - Your *shoulder to wrist* measurement is 60cm, so the sleeve is the length of your arm + 18cm
 - shoulder to shoulder is 52cm, neck circumference is 43cm, assuming a circle, that means neck diameter is 13.7cm. So, (52-13.7)/2 = 19.15 => This is the distance from the side of the neck to the shoulder. given that the neckline has some ease of its own, this seems a normal value.

So based on this, things seem normal to me. However, I have two remarks:

** Shoulder to wrist **

Your comparison still shows your sweater with a sleeve that's a lot shorter than the others. So why is that?  
These are the values used for the comparison for the `shoulderToWrist` measurement:

```
        usSize34: 670
        usSize36: 675
        usSize38: 680
        usSize40: 685
        usSize42: 690
        usSize44: 700
```

Your `shoulderToWrist` is 60cm, which means that in comparison to the range of sizes we compare to, your arms are shorter, thus your sleeve is shorter.

** Cuffs **

While investigating this, I realized there's an issue in Hugo with the `ribbingStretchFactor` option. This made the cuff length incorrect. Perhaps you didn't notice, but it's possible you mistook the incorrect cuff length (2.25cm) for the cuff width (8cm).

Long story short: This draft seems correct for the measurements you supplied. Please double-check your arm length, and while you're at it, your across back. You've updated it to a rather low value. In general the across back is about 4cm lower than the shoulder to shoulder.

I'll keep this issue open pending your feedback.

**Comment by joostdecock** [comment]
Oh, I haven't rolled out that cuff fix yet, but I will do so later today.

**Comment by JanGen** [comment]
Thank you for you analysis. I've mainly been thrown off by the comparison view. I'll try making a draft sleeve the next days to check up on the fit and will report back.

**Comment by joostdecock** [comment]
Just don't cut out the cuffs are they are currently. Wait for the fix for those. But they are just rectangles, so no big deal.

**Comment by JanGen** [comment]
Checked out the sleeve from a paper draft (without cuff) and it seems to check out fine. Seems indeed I'm just having short arms.
Concerning the cuff and waistband: The description of the option in the fit section and the parameter don't fit together either (seems to be inverted). The description of the ribbingStretchFactor reads "If you choose 90% (for example) for every 10cm of ribbing, we'll only draft 9cm.". However the default value is 20% (now 10%?) and the range limited to values of up to 30% (now 10%?).
I would actually opt to have it the way it is in the description, since that describes the method to actually determine the value.

**Comment by joostdecock** [comment]
We just (minutes ago) rolled out core v1.3.0 in production. This fixes the cuff issue, and indeed, the description and defaults have changed.

This change from 90% to 10% for the same amount of stretch has been implemented throughout all patterns, and in general users seem to find this approach more intuitive. This ribbing stretch in Hugo was overlooked at that time, and has been fixed now.

However, we still have to adapt the documentation in the frontend.

You should visit your draft page. You will/should see a message that your draft has updates (this is a new feature). If you re-draft it, it will be upgrade to version 1.3.0 and the ribbing issue will be fixed.

As for sleeve length. I am keeping this open for now as I have another person who raised a similar question and I promised to look into it tomorrow morning.

Thanks for your feedback :)

**Comment by joostdecock** [comment]
Sleeve was ok for that other user too. I'm closing this.
